cluster/juju/layers/kubernetes-master/config.yaml

```diff
@@ -40,3 +40,9 @@ options:
         runtime-config=batch/v2alpha1=true profiling=true
       will result in kube-apiserver being run with the following options:
         --runtime-config=batch/v2alpha1=true --profiling=true
+  authorization-mode:
+    type: string
+    default: "None"
+    description: |
+      Set the cluster's authorization mode. Allowed values are
+      "RBAC" and "None".
```

Comment:
multiple modes enabled at once are supported, recommended authorization modes are `Node,RBAC`

See:
* https://kubernetes.io/docs/admin/authorization/node/
* https://kubernetes.io/docs/admin/authorization/rbac/

Reply:
hmm... mapping `rbac` to `Node,RBAC` at https://github.com/kubernetes/kubernetes/pull/53820/files#diff-e6f5b987d0e976d7b76acd341abc74c0R1068 seems odd

Reply:
First of all, thank you for the prompt review!

Regarding this comment, we have revised our code so as the authorisation mode in this config to map exactly to the authorisation modes of Kubernetes. This means the user is expected to use the Node,RBAC to activate both the respective modes.